internal/pkg/docker/dockercompose/svc.go

```diff
@@ -0,0 +1,104 @@
+// Copyright Amazon.com, Inc. or its affiliates. All Rights Reserved.
+// SPDX-License-Identifier: Apache-2.0
+
+package dockercompose
+
+import (
+	"fmt"
+	"github.com/aws/copilot-cli/internal/pkg/manifest"
+	"github.com/compose-spec/compose-go/types"
+	"time"
+)
+
+func convertBackendService(service *types.ServiceConfig, port uint16) (*manifest.BackendServiceConfig, IgnoredKeys, error) {
+	image, ignored, err := convertImageConfig(service.Build, service.Labels, service.Image)
+	if err != nil {
+		return nil, ignored, fmt.Errorf("convert image config: %w", err)
```

Comment:
```suggestion
		return nil, nil, fmt.Errorf("convert image config: %w", err)
```
If it fails at some point we probably want to fail entirely and in that scenario we won't use `ignored` anymore.